File: src/main/kotlin/Garten.java
import java.util.ArrayList;

class Garden
{
    public Feld[][][] garten;
    private Akasha[] akashas=new Akasha[4];
    public Garden(int a, int b, int c)
    {
        garten=new Feld[a][b][c];
    }

    private void checkfeld(int a, int b, int c)
    {
        if(garten[a][b][c]==null) garten[a][b][c]=new Feld();
    }

    public void akashaHinzufuegen(Akasha a, int pos)
    {
        if(akashas[pos]==null)akashas[pos]=a;
        else System.out.println("Platz besetzt.");
    }

    public void akashaEntfernen(int pos)
    {
        if(akashas[pos]!=null)akashas[pos]=null;
    }
}

File: src/main/kotlin/Garten.kt
class Garten (var a: Int,var b: Int,var c: Int)
{
    var garten:Array<Array<Array<Feld?>>> = Array(a) { Array(b) { arrayOfNulls(c) } }
    fun init()
    {
        garten.forEach {
            it.forEach {
                for (i in 0..it.size-1)
                {
                    it[i]=Feld()
                }
            }
        }
    }

    private val akashas = arrayOfNulls<Akasha>(4)

    private fun checkfeld(a: Int, b: Int, c: Int) {
        if (garten[a][b][c] == null) garten[a][b][c] = Feld()
    }

    fun akashaHinzufuegen(a: Akasha?, pos: Int) {
        if (akashas[pos] == null) akashas[pos] = a else println("Platz besetzt.")
    }
}